docs/reference/aggregations/bucket/diversified-sampler-aggregation.asciidoc

```diff
@@ -83,10 +84,10 @@ Response:
 }
 --------------------------------------------------
 // TESTRESPONSE[s/\.\.\./"took": $body.took,"timed_out": false,"_shards": $body._shards,"hits": $body.hits,/]
-// TESTRESPONSE[s/1000/151/]
 // TESTRESPONSE[s/2.213/$body.aggregations.my_unbiased_sample.keywords.buckets.0.score/]
 
-<1> 1000 documents were sampled in total because we asked for a maximum of 200 from an index with 5 shards. The cost of performing the nested significant_terms aggregation was therefore limited rather than unbounded.
+<1> 151 documents were sampled in total because we asked for a maximum of 200 from an index with 5 shards. The cost of performing the
+nested significant_terms aggregation was therefore limited rather than unbounded.
```

Comment:
Indeed, it does not mean anything. I remove the "because we asked for a maximum of 200 from an index with 5 shards" because I think it is wrong. The returned response has doc_count: 151 and not 1000 (see the now removed `// TESTRESPONSE[s/1000/151/]` )